### PR TITLE
Add DBGL.app v0.79

### PR DESCRIPTION
**Pull request by rubenerd**
Adding the DOSBox Game Launcher (DBGL), a GUI frontend and configuration editor for DOSBox.

### CODE REVIEW

File: Casks/dbgl.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  sha256 '0a5aaf171f1a77f305564e5d952c24fe866e59c6db4678aee77cfde71172775c'
+    
+  url 'http://members.quicknet.nl/blankendaalr/dbgl/download/dbgl079.dmg'
+  name 'DBGL'
+  homepage 'http://members.quicknet.nl/blankendaalr/dbgl/'
+  license :gpl
+            
+  app 'DBGL.app'
+end
+
```
Comment (vitorgalvao) [inline Casks/dbgl.rb]: This empty line needs to go.

**Comment by vitorgalvao** [comment]
```
Casks/dbgl.rb:4:1: C: Trailing whitespace detected.
Casks/dbgl.rb:9:1: C: Trailing whitespace detected.
Casks/dbgl.rb:12:1: C: 1 trailing blank lines detected.
```

**Comment by RubenSchade** [comment]
Darn my Vim config and adding trailing whitespace. Corrected, sorry about that.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by RubenSchade** [comment]
Done. Thanks for your patience, this is my first PR on a public project, as you can probably tell!

**Comment by vitorgalvao** [comment]
Thank you for picking us. There’re still trailing white spaces, though, but I can fix those.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ad85c8821ac409c52f57c5af1954d7c6381ec266. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by RubenSchade** [comment]
Thanks Vítor, really appreciated :) I'll double check for these things for future contribs.

**Pull request closed without merging**